docs/DeveloperGuide.md

```diff
@@ -577,7 +576,7 @@ tab to view the newly added application. <br/>
   1b1. InternHunter displays an error message and informs the user that the index is out of bounds. <br />
   Use case resumes from step 1.
 
-#### Use case: UC09 - Delete an application
+#### Use case: UC011 - Delete an application
```

Comment:
Why aren't the application use cases refactored? Aren't they the same as company for delete, edit and view?

Reply:
yes as I mentioned in PR desc, I will update them after the feedback on for the the other items

Reply:
I have just updated the application portion!